routes/business.js

```diff
@@ -0,0 +1,11 @@
+// routes/newAndUpdates.js
+const express = require('express');
+const { registerBusiness, BusinessImageUpload, GetBusinessProfiles } = require('../controllers/businessController');
+const router = express.Router();
+const multer = require('multer');
+
+// Define a route to register business
+router.post('/register', registerBusiness);
```

Comment:
post routes should have authMiddleware. they should not be creating business without logging in